host-audit+measure-1/test.sh

```diff
@@ -36,7 +36,13 @@ mv "${TESTFILE}" "${TESTFILE}.renamed"
 auditlog_find ".*file=\"${TESTFILE}\" hash=\".*${filehash}\" .*" 1 10
 measurementlog_find "${SECURITYFS_MNT}" "${filehash}\s+${TESTFILE}\s+\$" 1
 
-auditlog_find ".*file=\"${TESTFILE}.renamed\" hash=\".*${filehash}\" .*" 0 10
+if [ -z "${IMA_TEST_UML}" ]; then
+  auditlog_find ".*file=\"${TESTFILE}.renamed\" hash=\".*${filehash}\" .*" 0 10
+else
+  # For some reason we find 1 entry.
```

Comment:
This is the blocker at the moment. Need to figure out why this happens in this case in UML.